.github/CONTRIBUTING.md

```diff
@@ -154,7 +154,7 @@ If any strange behaviour or exception is encountered, please submit a [bug repor
 
 ## Coding Standards
 
-See [OSS-Generics Coding Standards](https://github.com/oss-generic/process/blob/master/docs/CodingStandards.md)
+See [OSS-Generics Coding Standards](https://github.com/oss-generic/process/blob/master/docs/CodingStandards.adoc)
```

Comment:
Why not link directly to the C# document?

Reply:
This is better in case the project requires some non-C# code and the dev wants to know which standard to follow?

Reply:
That was what I thought too, even though unlikely...